app/styles/app.scss

```diff
@@ -2,7 +2,22 @@
 @import "components/results-page";
 @import "components/isp-jam-auth";
 @import "components/language-picker";
+@import 'ember-power-select/themes/bootstrap';
+@import 'ember-power-select';
 
+.ember-power-select {
```

Comment:
For every single select box anywhere in the application? If you are trying to style a particular select box, a more specific selector might be called for.

The idea here is that if we add further select boxes to this application later, they may not have the same use case. Global styles can be hard to unwind as an application grows. Like you saw with `.btn` recently, people usually end up custom-styling the *other* elements just to get back to the defaults!